lib/workers/repository/package-files.ts

```diff
@@ -26,42 +26,6 @@ export class PackageFiles {
     this.data.clear();
   }
 
-  /**
-   * Removes the last dependency/entry in the PackageFiles data
-   * i.e. the last line in the tobe generated detected dependency section
-   * @Returns true if anything that translates to a markdown written line was deleted
-   *          otherwise false is returned
-   */
-  private static pop(): boolean {
```

Comment:
this was moved to the end of the file as per conventional class structure (method declaration appearance order abstract -> public -> protected -> private)